addons/refuel/functions/fnc_refuel.sqf

```diff
@@ -29,6 +29,10 @@ _maxFuel = getNumber (configFile >> "CfgVehicles" >> (typeOf _target) >> QGVAR(f
     params ["_args", "_pfID"];
     _args params ["_source", "_sink", "_unit", "_nozzle", "_rate", "_startFuel", "_maxFuel", "_connectFromPoint", "_connectToPoint"];
 
+    if (_nozzle getVariable [QGVAR(isConnected), false]) exitWith {
```

Comment:
Nozzle gets deleted by [fnc_returnNozzle](https://github.com/acemod/ACE3/blob/master/addons/refuel/functions/fnc_returnNozzle.sqf#L48). So I'd rather check if _source is still connected.